<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QFormLayout" name="Layout">
      <item row="0" column="1">
       <widget class="QFrame" name="canvas">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>3</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
                                        height : 70%;
                                        border-style: solid;
                                        border-width: 1px;
                                        border-color: rgb(0, 0, 0);
                                        background: rgb(255,255,255);
                                    </string>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <widget class="QLabel" name="label_image">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>501</width>
           <height>471</height>
          </rect>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse</set>
         </property>
         <property name="buddy">
          <cstring></cstring>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <widget class="QPushButton" name="B_red">
          <property name="text">
           <string>빨강</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="B_blue">
          <property name="text">
           <string>파랑</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="B_black">
          <property name="text">
           <string>흑백</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="B_green">
          <property name="text">
           <string>초록</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>필터</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="image_bar">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;색조&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;명암&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;선명도&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="horizontalSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="horizontalSlider_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="horizontalSlider_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <property name="verticalSpacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>확대/축소</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="B_choose">
          <property name="text">
           <string>확대</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="B_choose_2">
          <property name="text">
           <string>축소</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>도구 툴</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="B_cut">
          <property name="text">
           <string>자르기</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="B_copy">
          <property name="text">
           <string>복제</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="B_areaChoose">
          <property name="text">
           <string>올가미</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="B_spoid">
          <property name="text">
           <string>스포이드</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="B_hbrush">
          <property name="text">
           <string>흐림 브러시</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>편집</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>오브젝트</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Open">
   <property name="text">
    <string>열기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>저장</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>실행 취소</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="text">
    <string>재실행</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="text">
    <string>잘라내기</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>복사</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>붙여넣기</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="text">
    <string>다른 이름으로 저장</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>끝내기</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>B_black</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>blackFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>661</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>B_red</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>redFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>661</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>B_green</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>greenFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>701</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>B_blue</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>blueFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>698</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>B_choose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>chooseFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>B_spoid</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>spoidFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>B_copy</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cloneFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>B_cut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cutFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>B_areaChoose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>boundFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>B_hbrush</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>brushFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>blackFunction()</slot>
  <slot>redFunction()</slot>
  <slot>greenFunction()</slot>
  <slot>blueFunction()</slot>
  <slot>chooseFunction()</slot>
  <slot>spoidFunction()</slot>
  <slot>cloneFunction()</slot>
  <slot>cutFunction()</slot>
  <slot>boundFunction()</slot>
  <slot>brushFunction()</slot>
 </slots>
</ui>
